crates/merod/src/cli/init.rs

```diff
@@ -165,6 +170,12 @@ impl InitCommand {
             .relayer_url
             .unwrap_or_else(defaults::default_relayer_url);
 
+        let protocol_value = match self.protocol.as_str() {
+            "near" => config::Protocol::Near,
+            "starknet" => config::Protocol::Starknet,
+            _ => config::Protocol::UnknownNetwork,
+        };
```

Comment:
if `Protocol` impls `FromStr`, you can move it to the args itself, so clap does the validation